app/src/main/kotlin/co/netguru/baby/monitor/client/data/communication/ClientEntity.kt

```diff
@@ -4,15 +4,16 @@ import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.Index
 import androidx.room.PrimaryKey
+import co.netguru.baby.monitor.client.feature.voiceAnalysis.VoiceAnalysisOption
 
 @Entity(
     tableName = "CLIENT_DATA",
     indices = [Index(value = ["firebase_key"], unique = true)]
 )
 data class ClientEntity(
     @ColumnInfo(name = "address") val address: String,
-    @ColumnInfo(name = "firebase_key") var firebaseKey: String
-) {
-    @PrimaryKey(autoGenerate = true)
-    var id: Int? = null
-}
+    @ColumnInfo(name = "firebase_key") var firebaseKey: String,
+    val voiceAnalysisOption: VoiceAnalysisOption = VoiceAnalysisOption.MachineLearning,
+    // There is only one parent handled right now
```

Comment:
Consider adding TODO to the comment

Reply:
I wouldn't see it as TODO as it isn't something that is waiting to be added. It's just not being handled and maybe never will be.